lighthouse-core/gather/gatherers/start-url.js

```diff
@@ -18,32 +18,11 @@ class StartUrl extends Gatherer {
   }
 
   executeFetchRequest(driver, url) {
-    return new Promise((resolve, reject) => {
-      let requestId;
```

Comment:
@wardpeet I think all this was only necessary in a previous version during review when you needed `requestId` IIRC, was there another reason you can remember we didn't just use evaluateAsync on `fetch`?

Reply:
I cannot recall why I needed it but my first version had exact the same fetch as you have now. I believe @brendankenny  wanted to give a good error message in a follow up but besides that I don't really know why the network dance is in there.